File: app/src/main/java/de/bahnhoefe/deutschlands/bahnhofsfotos/model/ProblemReport.java
package de.bahnhoefe.deutschlands.bahnhofsfotos.model;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class ProblemReport {

    String countryCode;
    String stationId;
    String comment;
    ProblemType type;
    Long photoId;
    Double lat;
    Double lon;

}

File: app/src/main/kotlin/de/bahnhoefe/deutschlands/bahnhofsfotos/model/ProblemReport.kt
package de.bahnhoefe.deutschlands.bahnhofsfotos.model

data class ProblemReport @JvmOverloads constructor(
    var countryCode: String,
    var stationId: String,
    var comment: String? = null,
    var type: ProblemType,
    var photoId: Long? = null,
    var lat: Double? = null,
    var lon: Double? = null,
    var title: String? = null,
)